modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -0,0 +1,12 @@
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	//Non cat-people smash into the ground
+	if(!isfelinid(src) || !istajaran(src))
```

Comment:
If I didn't catch this, it would have never worked. Because Felinids aren't Tajaran, and vice-versa.
```suggestion
	if(!isfelinid(src) && !istajaran(src))
```